components/shapediver/parameter/ParameterSliderComponent.tsx

```diff
@@ -125,7 +128,11 @@ export default function ParameterSliderComponent(
 							step={step}
 							onChange={(v) => setValue(round(definition, v))}
 							onChangeEnd={(v) =>
-								handleChange(round(definition, v), 0)
+								handleChange(
+									round(definition, v),
+									undefined,
```

Comment:
@LemToUp let's keep this to 0, otherwise there will be a delay

Reply:
@snabela that's uncomfortable for controls from keyboard - update happens immediately.

Reply:
@LemToUp do you mean updates from the NumberInput below, correct? Here, for the slider, we used `0` previously as well.

Reply:
@LemToUp I tested again, it should really be 0 for the slider (as it was before)

Reply:
@snabela done.